src/CommandInfoList.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandInfoList {
+    private List<String> commandInfoList;
+    private String addCommand = "Чтобы добавить нового пользователя, введи команду => 'add'";
```

Comment:
Each command can encapsulate information about what the command are doing. In such case you don't need this object, just List<Command> commands would be enough